src/initialization.js

```diff
@@ -53,8 +53,17 @@ var initialization = {
                     });  
                 }
 
-                helpers.loadScript('https://unpkg.com/@optimizely/optimizely-sdk/dist/optimizely.browser.umd.min.js', 
-                helpers.loadScript('https://cdn.optimizely.com/datafiles/' + settings.projectId + '.json/tag.js', instantiateFSClient));
+                helpers.loadScript(
+                    'https://unpkg.com/@optimizely/optimizely-sdk@3.5.0/dist/optimizely.browser.umd.min.js',
```

Comment:
Would it be appropriate to move urls into a constant?

Reply:
it would.  do you think it should just be this one, and not worry about piece-mealing the second one below to be like?

ie avoiding something like (excuse the not so terseness of the following since we need to do es5):
var dataFileDomain = https://cdn.optimizely.com/datafiles/
var dataFileURLending = '.json/tag.js'
```
helpers.loadScript(
    dataFileDomain + settings.projectId + dataFileURLending,
    instantiateFSClient
);
```

Reply:
Yeah, that works with me. Although you probably should use `dataFilePrefix` instead of `dataFileDomain`